### ISSUE TITLE
Convert sections in monospace font to Markdown fenced code blocks

### ISSUE THREAD

**Issue by Cito** [akosbalasko/yarle#128]
Yarle already has a rule that recognizes the grey Evernote code blocks (which use the CSS style "-en-codeblock") and converts them to Markdown fenced code block.

Unfortunately, these Evernote code blocks were only available in newer Versions of Evernote. Therefore we used to format code blocks with monospace fonts in the earlier days. It would be great if Yarle could recognize such sections with monospace fonts and convert them to Markdown fenced code blocks as well.

Yarle would need to inspect the CSS style of the blocks to do that. If there is a "font-family: monospace" (or other well-known monospaced fonts like Courierl), and it is not mixed with other fonts, then this should be interpreted as a code block.

**Comment by Cito** [comment]
A test note could look like this:
```
This should be a (single) code block:
<div align="left"><font color="#010101" face="Courier New" size="2">foo</font></div>
<div align="left"><font color="#010101" face="Courier New" size="2">bar</font></div>
<div align="left"><font color="#010101" face="Courier New" size="2">baz</font></div>
This should be another one:
<div style="font-family:monospace">foo bar</div>
This should not be one:
<div style="font-family:monospace"><span style="font-family:serif">foo bar</span></div>
```

**Comment by akosbalasko** [comment]
Implemented, merged, published in 3.4.0. 
@Cito , thank you for your contribution!
